netsim/extra/bgp.session/defaults.yml

```diff
@@ -205,6 +208,7 @@ bgp:
         max_value: 10
       as_override: bool
       default_originate: bool
+      description: str
```

Comment:
Could consider to put this ```description``` attribute on the interface. Could be useful for other modules besides bgp?

I've sometimes used the ```name``` attribute of links to document test cases, but there is nothing yet for interfaces

Reply:
> Could consider to put this `description` attribute on the interface. Could be useful for other modules besides bgp?

I will probably make this a standard BGP attribute. It's the stuff that's configured on the BGP neighbor statement, and is not necessarily the same thing you'd want to see on an interface.

> I've sometimes used the `name` attribute of links to document test cases, but there is nothing yet for interfaces

Shall we make the "name" an interface attribute as well and check whether it exists before recreating it?